src/frontend/src/sass/_form.scss

```diff
@@ -57,9 +57,19 @@
   margin: 0;
 }
 
+.search-container-wrapper {
+  display: flex;
+  align-items: center;
+  margin-top: 15px;
+  width: 100%;
+  label {
+  display: none;
+  }
+}
+
 .search-container {
   min-width: 450px;
-  width: 42%; /* based on branding eye-balling */
+  width: 0; /* based on branding eye-balling */
```

Comment:
width: 0?

Reply:
Also, when we change stuff that has attached comments, can we change or remove the comments accordingly?